tools/cloud-build/daily-tests/ansible_playbooks/spack.yml

```diff
@@ -14,18 +14,11 @@
 
 ---
 
-- name: Wait for startup script to complete
-  become: true
-  wait_for:
-    path: /var/log/messages
-    search_regex: '.*{{ login_node }}.*startup-script exit status ([0-9]+)'
-    timeout: 7200
-    state: present
```

Comment:
According to [ansible docs](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/wait_for_module.html#parameter-state), when used for a file/search string (which is what we are doing), `present` has the same effect as the default `started`.

Reply:
I agree with the logic. I'd raise at a "nit" level that I prefer being explicit for state for clarity and for possibility of default behavior changes. This is probably not one of those cases.

Reply:
Ack.